src/main/java/gui/EERPromptGUI.java

```diff
@@ -203,4 +209,14 @@ private void helperForDisplay(int type){
             }
         }
     }
+
+    private void createUIComponents() {
+        try{
+            headerImg = ImageIO.read(GUIFormatConstants.eerAnalyzerImgFile);
+        }catch (IOException ex){
+            System.out.println("File pathway was not found");
```

Comment:
this is a very small thought, but wondering if "the File pathway was not found" we should put it into the constants class as well? just to make it organized?